connect/build.boot

```diff
@@ -0,0 +1,52 @@
+(set-env!
+ :resource-paths #{"resources"}
+ :dependencies '[[cljsjs/boot-cljsjs "0.9.0" :scope "test"]])
+
+(require '[cljsjs.boot-cljsjs.packaging :refer :all]
+         '[boot.core :as boot]
+         '[boot.tmpdir :as tmpd]
+         '[clojure.java.io :as io]
+         '[boot.util :refer [sh]])
+
+(def +lib-version+ "0.6.6")
+(def +version+ (str +lib-version+ "-0"))
+
+(task-options!
+ pom {:project     'cljsjs/connect
```

Comment:
I think name should be `cljsjs/0xproject-connect` or something similar. We try to reflect npm names and npm Connect package is something completely different.